photospp-toolfile.spec

```diff
@@ -1,4 +1,4 @@
-### RPM external photospp-toolfile 3.56
+### RPM external photospp-toolfile 2.0
```

Comment:
There is no such rule in cmsdist - see other toolfiles. The content of toolfiles is rarely changed.